src/blockchain_db/lmdb/db_lmdb.h

```diff
@@ -39,15 +39,20 @@
 namespace cryptonote
 {
 
+typedef struct mdb_block_info
+{
+  uint64_t bi_timestamp;
+  uint64_t bi_coins;
+  size_t bi_size;
```

Comment:
Yes, this broke 32-bit compatibility. Fixed already in following commit https://github.com/monero-project/bitmonero/pull/794/commits/bef1bbf5ea8ea6055046363fcb326e9fa4e68393